### DISCUSSION TITLE
Short circuit analysis

### DISCUSSION THREAD

**Discussion by miek770** [SanPen/GridCal#184] [Q&A] [answered]
Hi @SanPen , I have a few questions about the short circuit capabilities and didn't feel they justified issues (at least not yet); I just found out that GitHub discussions were a thing!

I'm currently working on a small tutorial for short circuit calculations in Python and would like to include an example with GridCal. My intent is to reproduce the example in Figure 4 from this IEEE transaction: [Short Circuit ABC](https://arcadvisor.com/files/ShortCircuitABC.pdf)

I have no issue reproducing it by hand (obviously), and with ETAP (I get 11 kA, as expected). I tried with GridCal through the CLI and then the UI and get a significantly higher result (586 MVA at the 12 kV bus, while 228 MVA is expected).

To investigate, I checked the tests and saw that the [short circuit test](https://github.com/SanPen/GridCal/blob/master/src/tests/test_short_circuit.py) doesn't compare the result with expected values; I'm not familiar with IEEE's standard grids, do you have the expected results and could we compare it with the results in the test?

If not, would you be open to considering new test cases with data from ETAP, PSS/E or EasyPower?

Would you consider both ANSI/IEEE and IEC short circuit cases?

**Comment by SanPen** [comment, marked as answer]
Hi Michel!

I am completely open to more test cases.

In fact I'm preparing the release of GridCal 5 (on the devel branch) and this is definitely a thing to consider.

Could you please upload the gridcal file?

Just to warn you, the unbalanced short circuit test is failing...I'll investigate it so we can get this done.

BR,
Santiago

**Comment by saulfeza** [comment]
Hi everyone,
It is still possible to access this tutorial? I have big issues about designing and calulating short-circuit currents .

Saul Ferrer

**Reply by miek770** [reply]
Hi @saulfeza , I did publish [a brief blog post on this subject in 2023](https://codecurrents.blog/article/2023-09-16) but it appears my motivation exceeded my dedication... Hope this helps.
